src/Razor/benchmarks/Microsoft.AspNetCore.Razor.Microbenchmarks/Resources/Telerik/Kendo.Mvc.Examples.project.razor.json

```diff
@@ -1,10 +1,12 @@
 {
-  "__Version": 6,
+  "__Version": 7,
   "ProjectKey": "C:\\Users\\admin\\location\\Kendo.Mvc.Examples\\obj\\Debug\\net7.0\\",
   "FilePath": "C:\\Users\\admin\\location\\Kendo.Mvc.Examples\\Kendo.Mvc.Examples.csproj",
   "Configuration": {
     "ConfigurationName": "MVC-3.0",
     "LanguageVersion": "7.0",
+    "UseRoslynTokenizer": true,
+    "CSharpLanguageVersion": 1100,
```

Comment:
I realize that the shape of the JSON payload is really irrelevant, but it does _feel_ better that 
"CSharpLanguageVersion" is hanging off of "Configuration" on line 9 rather than line 159531. 😁